xapian-applications/omega/docs/omegascript.rst

```diff
@@ -542,6 +542,15 @@ $set{OPT,VALUE}
 	Similarly, ``XFOO:stemmer`` specifies the stemmer to use for field
 	``XFOO``, with ``stemmer`` providing a default.
 
+$seterror{error_msg}
+	set error message for the current execution, which can also be looked
+	up using ``$error``.
+
+	Using ``$seterror`` error early in template prevent running query.
+	Saves us from executing query unnecessarily.
+
+	You can use ``$seterror`` when the user enters a wrong parameter in the search.
```

Comment:
We should make clear that this is just an example of how you can use `$seterror`.